### PR TITLE
DS3, Docs: Add downpatching instructions to Dark Souls III setup guide

### PR DESCRIPTION
**Pull request by Samonella7**
## What is this fixing or adding?
Added instructions for downpatching Dark Souls III, which is required by the archipelago mod

## How was this tested?
I have tested that the mod works using the linked downpatching method

### CODE REVIEW

File: worlds/dark_souls_3/docs/setup_en.md [hunk 1]
```diff
@@ -11,27 +11,34 @@
...
 add it at the root folder of your game (e.g. "SteamLibrary\steamapps\common\DARK SOULS III\Game")
 
 ## Joining a MultiWorld Game
 
-1. Run DarkSoulsIII.exe or run the game through Steam
+1. Run Steam in offline mode, both to avoid being banned and to prevent Steam from updating the game files
+1. Launch Dark Souls III
 2. Type in "/connect {SERVER_IP}:{SERVER_PORT} {SLOT_NAME}" in the "Windows Command Prompt" that opened
 3. Once connected, create a new game, choose a class and wait for the others before starting
 4. You can quit and launch at anytime during a game
```
Comment (black-sliver) [inline worlds/dark_souls_3/docs/setup_en.md:39]: Please renumber properly

**Review by black-sliver — CHANGES_REQUESTED** [review]

**Comment by black-sliver** [comment]
@Marechal-L

File: worlds/dark_souls_3/docs/setup_en.md (hunk 1 above)
Comment (Samonella7) [reply worlds/dark_souls_3/docs/setup_en.md]: my mistake, fixed

File: worlds/dark_souls_3/docs/setup_en.md [hunk 2]
```diff
@@ -21,9 +21,7 @@ This client has only been tested with the Official Steam version of the game at
 
 ## Downpatching Dark Souls III
 
-Download the 1.15 crash-fix executable from the [speedsouls wiki](https://wiki.speedsouls.com/darksouls3:Crash_Fix) and use it to replace DarkSoulsIII.exe in your installation. This modified executable also fixes a memory leak that may otherwise cause the game to crash.
-
-An alternate method is to download version 1.15 from the Steam console. Follow instructions from the [speedsouls wiki](https://wiki.speedsouls.com/darksouls3:Downpatching). Your download command, including the correct depot and manifest ids, is "download_depot 374320 374321 4471176929659548333"
+Follow instructions from the [speedsouls wiki](https://wiki.speedsouls.com/darksouls3:Downpatching). Your download command, including the correct depot and manifest ids, will be "download_depot 374320 374321 4471176929659548333"
```
Comment (black-sliver) [inline worlds/dark_souls_3/docs/setup_en.md:24]: Should it maybe mention the version in addition to the IDs so people can sanity check?

Comment (Samonella7) [reply worlds/dark_souls_3/docs/setup_en.md]: It says 1.15 above, but you're right, it's much nicer to specify again in the down patching section. done

File: worlds/dark_souls_3/docs/setup_en.md [hunk 3]
```diff
@@ -11,27 +11,32 @@
...
 ## General Concept
 
+<span style="color:tomato">
+**This mod can ban you permanently from the FromSoftware servers if used online.** 
+</span>
 The Dark Souls III AP Client is a dinput8.dll triggered when launching Dark Souls III. This .dll file will launch a command 
 prompt where you can read information about your run and write any command to interact with the Archipelago server.
 
-## Installation Procedures
+This client has only been tested with the Official Steam version of the game at version 1.15. It does not matter which DLCs are installed. However, you will have to downpatch your Dark Souls III installation from current patch, which is 1.15.2.
```
Comment (black-sliver) [inline worlds/dark_souls_3/docs/setup_en.md:20]:
```suggestion
This client has only been tested with the Official Steam version of the game at version 1.15. It does not matter which DLCs are installed. However, you will have to downpatch your Dark Souls III installation from current patch.
```

**Review by black-sliver — APPROVED** [review]
Didn't hear back from Marechal-L, but got the one suggestion and the OK from br00ty.

**Pull request merged**